docs/learn/expert-guides/deploy-up-with-lsp23.md

```diff
@@ -147,94 +151,87 @@ When deploying your Universal Profile, you might want to initialize it with some
 
 #### Create the LSP3 Metadata
 
-The LSP3 Metadata is a JSON object that contains information about the Universal Profile. For more information on the LSP3 Metadata, please refer to the [LSP3 Profile Metadata specification](../../standards/universal-profile/lsp3-profile-metadata.md). For the sake of simplicity, we will not create the JSON object in this guide but use random bytes instead:
+The LSP3 Metadata is a JSON object that contains information about the Universal Profile. For more information on the LSP3 Metadata, please refer to the [LSP3 Profile Metadata specification](../../standards/universal-profile/lsp3-profile-metadata.md). For the sake of simplicity, we will use one that we previously created and pushed to IPFS:
 
 ```typescript
 async function main() {
   // previous code
 
-  const lsp3DataKey = ERC725YDataKeys.LSP3.LSP3Profile;
-  const lsp3DataValue = ethers.randomBytes(32); // this is just a random value for the LSP3Profile
-}
-```
-
-#### Create the Universal Receiver data key
-
-```typescript
-async function main() {
-  // previous code
+  // instantiate the erc725 class
+  const erc725 = new ERC725([
+    ...LSP6KeyManagerSchemas,
+    ...LSP3ProfileMetadataSchemas,
+    ...LSP1UniversalReceiverDelegateSchemas,
+  ]);
 
-  const universalReceiverDataKey =
-    ERC725YDataKeys.LSP1.LSP1UniversalReceiverDelegate;
+  const lsp3DataValue = {
+    verification: {
+      method: 'keccak256(utf8)',
+      data: '0x6d6d08aafb0ee059e3e4b6b3528a5be37308a5d4f4d19657d26dd8a5ae799de0',
+    },
+    url: 'ipfs://QmPRoJsaYcNqQiUrQxE7ajTRaXwHyAU29tHqYNctBmK64w',
```

Comment:
Maybe would be good to mention in comment here to replace with the actual user values?